pkg/apis/awsprovider/v1alpha1/types.go

```diff
@@ -442,3 +442,9 @@ type Instance struct {
 	// The tags associated with the instance.
 	Tags map[string]string `json:"tags,omitempty"`
 }
+
+// TODO move somewhere else?
+const (
+	AnnotationClusterInfrastructureReady = "aws.cluster-api.sigs.k8s.io/infrastructure-ready"
```

Comment:
Should we follow the api group name? `aws.cluster-api.sigs.k8s.io` -> `aws.cluster.sigs.k8s.io` ?

Reply:
Sure

Reply:
I like the idea of using `cluster-api`, since it is more directly tied to the project name, but I don't think `-` can be used in api groups, so maybe `cluster` is better for consistency.

Reply:
I'd say if we want to make it future-ready we can also use `aws.infrastructure.cluster.sigs.k8s.io` 😄